pyanaconda/modules/network/firewall/firewall_interface.py

```diff
@@ -0,0 +1,121 @@
+    def FirewallKickstarted(self) -> Bool:
+        """Was the firewall command present in the input kickstart ?"""
+        return self.implementation.firewall_seen
+
+    @property
+    def UseSystemDefaults(self) -> Bool:
+        """Should system defaults be used for firewall configuration ?
+
+        This effectively means Anaconda will not configure Firewall in any way
+        and installed system defaults will be used.
+        """
+        return self.implementation.use_system_defaults
+
+    @emits_properties_changed
+    def SetUseSystemDefaults(self, use_system_defaults: Bool):
+        """Set if system defaults should be used for the firewall configuration."""
+        self.implementation.set_use_system_defaults(use_system_defaults)
+
+    @property
+    def FirewallEnabled(self) -> Bool:
+        """Should firewall be enabled ?"""
+        return self.implementation.firewall_enabled
+
+    @emits_properties_changed
+    def SetFirewallEnabled(self, firewall_enabled: Bool):
+        """Set if the firewall should be enabled or disabled."""
+        self.implementation.set_firewall_enabled(firewall_enabled)
+
+    @property
+    def EnabledPorts(self) -> List[Str]:
+        """List of ports to be allowed through the firewall."""
+        return self.implementation.enabled_ports
+
+    @emits_properties_changed
+    def SetEnabledPorts(self, enabled_ports: List[Str]):
+        """Set the list of ports to be allowed thorough the firewall.
+
+        :param enabled_ports: a list of ports to be enabled
+        """
+        self.implementation.set_enabled_ports(enabled_ports)
+
+    @property
+    def Trusts(self) -> List[Str]:
+        """List of trusted devices to be allowed through the firewall."""
+        return self.implementation.trusts
+
+    @emits_properties_changed
+    def SetTrusts(self, trusts: List[Str]):
+        """Set the list of trusted devices to be allowed through the firewall.
+
+        :param trusts: a list of trusted devices
+        """
+        self.implementation.set_trusts(trusts)
+
+    @property
+    def EnabledServices(self) -> List[Str]:
+        """List of services to be allowed through the firewall."""
+        return self.implementation.enabled_services
+
+    @emits_properties_changed
+    def SetEnabledServices(self, enabled_services: List[Str]):
+        """Set the list of services to be allowed through the firewall.
+
+        :param enabled_services: a list of services to be enabled
+        """
+        self.implementation.set_enabled_services(enabled_services)
+
+    @property
+    def DisabledServices(self) -> List[Str]:
+        """List of services to be explicitly disabled on the firewall."""
+        return self.implementation.disabled_services
+
+    @emits_properties_changed
+    def SetDisabledServices(self, disabled_services: List[Str]):
+        """Set the list of services to be explicitly disabled on the firewall.
+
+        :param enabled_services: a list of services to be enabled
```

Comment:
enabled -> disabled

Reply:
The documentation of a parameter is still not fixed.

Reply:
Fixed.